### PR TITLE
feature/SE-2341.TransactionsForDifferentFileTypes

### PR DESCRIPTION
**Pull request by fox8090**
Added three other file types. I think the txt file is similar to a typical use case but feel free to let me know otherwise.

### CODE REVIEW

File: examples/lusid/transactions/upload-transactions-from-csv.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
+-- ============================================================
+
+-- Description:
+-- 1. In this query, we run an ETL process on some transactions.
+-- 2. First, we load a CSV file of transactions from Drive.
```
Comment (slemasne) [inline examples/lusid/transactions/upload-transactions-from-csv.sql:5]: typo:

```
we load a CSV
```

File: examples/lusid/transactions/upload-transactions-from-csv.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+-- Extract transaction data from LUSID Drive
+@txn_data = 
+use Drive.Csv
+--file=/luminesce-examples/transactions.csv
+enduse;
+
+-- Set variables for the portfolio's scope and code
+@@portfolio_scope =
+select 'IBOR';
```
Comment (slemasne) [inline examples/lusid/transactions/upload-transactions-from-csv.sql:19]: can we put this all on one line?
Same for other files.

Comment (fox8090) [reply examples/lusid/transactions/upload-transactions-from-csv.sql]: Changes made.

File: examples/lusid/transactions/upload-transactions-from-excel.sql [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+-- 3. Next, we transform the shape of the transaction data.
+-- 4. Finally we upload the transaction data into LUSID.
+
+-- ============================================================
+
+-- Extract transaction data from LUSID Drive
+@txn_data = 
+use Drive.Excel
+--file=/luminesce-examples/transactions.xlsx
+--worksheet=Sheet 1
```
Comment (DavidWassellFinbourne) [inline examples/lusid/transactions/upload-transactions-from-excel.sql:15]: Does the excel workbook have more than 1 sheet? If not this is redundant.

**Review by DavidWassellFinbourne — COMMENTED** [review]
Looks good apart from my 1 comment.

File: examples/lusid/transactions/upload-transactions-from-excel.sql (hunk 3 above)
Comment (fox8090) [reply examples/lusid/transactions/upload-transactions-from-excel.sql]: It does only have one sheet. Code amended.

**Pull request merged**